drivers/ethernet/Kconfig.w5500

```diff
@@ -35,3 +35,13 @@ config ETH_W5500_TIMEOUT
 	  Given timeout in milliseconds. Maximum amount of time
 	  that the driver will wait from the IP stack to get
 	  a memory buffer before the Ethernet frame is dropped.
+
+config ETH_W5500_INT_TIMEOUT
+	int "Interrupt gpio timeout"
```

Comment:
I suggest to add the resolution of the timeout so something like `Interrupt gpio timeout in ms`